lib/util.js

```diff
@@ -153,13 +153,22 @@ util = {
         !callback && _.isFunction(options) && (callback = options, options = {});
 
         var postmanApiKey = _.get(options, 'postmanApiKey'),
-            headers = { 'User-Agent': USER_AGENT_VALUE };
+            headers = { 'User-Agent': USER_AGENT_VALUE },
+            error;
 
         // build API URL if `location` is a valid UID and api key is provided.
         // Fetch from file in case a file with valid UID name is present.
-        if (!fs.existsSync(location) && POSTMAN_API_PATH_MAP[type] && postmanApiKey && UID_REGEX.test(location)) {
-            location = `${POSTMAN_API_URL}/${POSTMAN_API_PATH_MAP[type]}/${location}`;
-            headers[API_KEY_HEADER] = postmanApiKey;
+        if (!fs.existsSync(location) && POSTMAN_API_PATH_MAP[type] && UID_REGEX.test(location)) {
+            if (postmanApiKey) {
```

Comment:
I would suggest bailing out early if the `postmanApiKey` key is missing... to avoid if..else branch.